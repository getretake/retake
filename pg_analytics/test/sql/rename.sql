CREATE TABLE t (a int, b text) USING parquet;
INSERT INTO t VALUES (1, 'a'), (2, 'b'), (3, 'c');
ALTER TABLE t RENAME TO s;
SELECT * FROM s;
SELECT * FROM t;
DROP TABLE t;
DROP TABLE s;
