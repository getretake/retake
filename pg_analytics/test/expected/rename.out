CREATE TABLE t (a int, b text) USING deltalake;
INSERT INTO t VALUES (1, 'a'), (2, 'b'), (3, 'c');
ALTER TABLE t RENAME TO s;
SELECT * FROM s;
 a | b 
---+---
 3 | c
 2 | b
 1 | a
(3 rows)

SELECT * FROM t;
ERROR:  relation "t" does not exist
LINE 1: SELECT * FROM t;
                      ^
DROP TABLE t;
ERROR:  relation "t" does not exist
DROP TABLE s;
